pkg/upgrade/consts.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022 NVIDIA CORPORATION & AFFILIATES
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+const (
+	UpgradeStateAnnotation     = "nvidia.com/ofed-upgrade-state"
+	PodTemplateGenerationLabel = "pod-template-generation"
+
+	Unknown         = ""
```

Comment:
These are the different upgrade states a node may be in right ?

can you add a comment explaining it  ? 

maybe prefix it with UpgradeState<the-state> ?